scripts/start-deployMagmaMaintained

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# shellcheck source=start-utils
+. "${SCRIPTS:-/}start-utils"
+isDebugging && set -x
+
+: "${FORGE_VERSION:=${FORGEVERSION:-RECOMMENDED}}"
+: "${MAGMA_MAINTAINED_TAG:=}"
+
+resolveVersion
+
+if ! downloadUrl="https://github.com/magmamaintained/Magma-${VERSION}/releases/download/${MAGMA_MAINTAINED_TAG}/magma-${VERSION}-${FORGE_VERSION}-${MAGMA_MAINTAINED_TAG}-server.jar"; then
```

Comment:
This if-statement doesn't look right. You have a variable assignment from a literal as a condition. Did you confirm this runs?

Reply:
Thank you for your reviewing!

I have tested that the server is correctly downloaded and started. But I have misunderstood the meaning of this `if` condition. So, I removed that and just replaced it into plain assignment statement with empty variable validation.

Reply:
@itzg @aruneko Actual download url is https://github.com/magmamaintained/Magma-1.12.2/releases/download/88659fb/Magma-1.12.2-88659fb-server.jar

Please remove FORGE_VERSION from the script

Reply:
Adjusted in #2562

Reply:
https://github.com/itzg/docker-minecraft-server/pull/2562#issuecomment-1870781760
download link was correct for >1.18